src/core/lib/gprpp/arena.cc

```diff
@@ -58,7 +53,8 @@ Arena* Arena::Create(size_t initial_size) {
        GPR_CACHELINE_SIZE % GPR_MAX_ALIGNMENT == 0)
           ? GPR_CACHELINE_SIZE
           : GPR_MAX_ALIGNMENT;
-  return new (gpr_arena_malloc<alignment>(alloc_size)) Arena(initial_size);
+  void* storage = gpr_malloc_aligned(alloc_size, alignment);
+  return new (storage) Arena(initial_size);
```

Comment:
The two lines above could be combined again.

Reply:
Done.